### DISCUSSION TITLE
What do you think is the answer?

### DISCUSSION THREAD

**Discussion by batuhangithub** [enzifiri/GitHub-Achievements#376] [Q&A] [answered]
You want to reverse the text stored in a text file (example.txt) and save it back to the same file, so that the content is entirely reversed. How would you accomplish this task?

**Comment by muslumhanozturk** [comment, marked as answer]
# Read the file to retrieve its content
with open('example.txt', 'r') as file:
    content = file.read()

# Reverse the text
reversed_content = content[::-1]

# Write the reversed content back to the file
with open('example.txt', 'w') as file:
    file.write(reversed_content)
